src/tests/base_test.py

```diff
@@ -182,10 +182,10 @@ def setUp(self):
             "Content-Type": "application/json"
         }
 
-        # test countries
-        self.kenya = Country(name='Kenya')
-        self.uganda = Country(name='Uganda')
-        self.nigeria = Country(name='Nigeria')
+        # test centers
+        self.kenya = Center(name='Kenya')
```

Comment:
How about including Center as a different column instead and retaining the old Country column?